### ISSUE TITLE
Initial Screen

### ISSUE THREAD

**Issue by veronicamontoya** [Protocentral/protocentral-healthypi-v3#17]
Hi, I am having trouble with my Healthy Pi. I connected it properly, made sure everything was fine and updated, but once i copy the code on the terminal window, agree and reboot nothing shows up on the homescreen.  What can i do?

**Comment by amafj** [comment]
I'm facing the same issue! did you solve it?

**Comment by veronicamontoya** [comment]
1. Download processing on the raspberry
2. Download straight from the pi all the documents that appear on the
website
3. Once this is done, connect the pi through the usb cable that came with
it, do not connect it through the pins.
4. In the downloaded files, look for the file named healthy pi master (or
something like that). There is a file called gui, you select it and inside
there is a document called gui.pce and you run it through the processing
program.

**Comment by amafj** [comment]
I've downloaded "processing" earlier, but  he problem is one I unzip the file nothing is extracted!

**Comment by veronicamontoya** [comment]
Yes, same to me. I researched and found processing is a program, so I
downloaded the program in the raspberry. Then, when I unzipped the files I
went to the healthy master file. Once in there, I opened another file
called GUI, and in there is a doc called gui.pce. The gui.pce file is the
one you need to run through processing in order for the window to show up.

**Comment by amafj** [comment]
dude are you on whatsapp or can I get ur email?

**Comment by amafj** [comment]
because this thing is driving me crazy

**Comment by veronicamontoya** [comment]
+504 9535 7977

**Comment by archie020** [comment]
@veronicamontoya Kindly provide the following information for us to assess the issue:
1) Which Operating system are you using?
2) What method have you tried to install?
3) Have you purchased only the board or the entire unit including display?
4) Do attach the images or screenshots (for any errors)

**Comment by archie020** [comment]
@amafj Kindly provide the following information for us to assess the issue:
1) Which Operating system are you using?
2) What method have you tried to install?
3) Have you purchased only the board or the entire unit including display?
4) Do attach the images or screenshots (for any errors)

**Comment by amafj** [comment]
1- I'm using rasbian.
2- I've tried both methods, the advanced worked but I don't get any output. only a GUI.
3- Board and sensors only.
![66c7dcd2-d219-4d95-a47c-a5122ed0f7ea](https://user-images.githubusercontent.com/45542950/49530427-21373980-f8b0-11e8-9419-d3152f26b23b.JPG)
![d6a6cf86-f589-4766-9ebf-9fefd1b6d550](https://user-images.githubusercontent.com/45542950/49530453-2c8a6500-f8b0-11e8-91b1-176ec6616c9a.JPG)


This is the output for running "healthypi3_gui_tcp.pde" codes.

Please help, the board is useless so far!!
